### PR TITLE
feat: Update curriculum links, formatting

### PR DESCRIPTION
**Pull request by catheraaine**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
 
 * [CodeSchool: Try Git][]
 * [CodeSchool: Git Real][]
-* [CodeSchool: Git Real 2][] - Levels 1, 2, 4, 6
+* [CodeSchool: Git Real 2][]
```
Comment (catheraaine) [inline index.md]: I removed the levels as CodeSchool doesn't allow you to skip lessons. In order to do 4 and 6, you have to have completed 3 and 5.

File: index.md [hunk 2]
```diff
@@ -57,36 +57,40 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
 * Introduction to Jasmine
 * [FizzBuzz Kata - Javascript][] *with unit tests*
-* [CodeSchool: CoffeeScript][]
-* [CodeSchool: jQuery Air: First Flight][]
-* [CodeSchool: jQuery Air: Captain's Log][]
-* Frontend project JavaScript review (CSS to be rewritten)
-* Frontend project final review and launch!
+* [Javascript Road Trip Part I](https://www.codeschool.com/courses/javascript-road-trip-part-1)
+* [Javascript Road Trip Part II](https://www.codeschool.com/courses/javascript-road-trip-part-2)
+* [Optional: Javascript Road Trip Part III](https://www.codeschool.com/courses/javascript-road-trip-part-3)
```
Comment (catheraaine) [inline index.md]: Removed JQuery / CoffeeScript completely. Added some JS. Part I is ~3hours, II is ~5hours, and III ~8 hours. I haven't done III myself.

File: index.md [hunk 3]
```diff
@@ -57,36 +57,40 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
+* Frontend Project - Review: JavaScript
```
Comment (catheraaine) [inline index.md]: I am not sure the Frontend Project actually has JS in its current form, @zastrow ?

File: index.md [hunk 4]
```diff
@@ -101,14 +105,15 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
 * Create a to-do app
-* Kick off team project
-* Complete and launch team project
-* Intro to sparkbox/project-init
-* Kick off Pet Project
+* Apprentice Project: Kick off
+* Apprentice Project: Work
+* Apprentice Project: Launch :rocket:
+* Intro to `sparkbox/project-init`
+* Pet Project: Kick off
```
Comment (catheraaine) [inline index.md]: 115 / 116 Are not something I remember, can they be removed or clarified?

File: index.md [hunk 5]
```diff
@@ -57,36 +57,40 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
 * [FizzBuzz Kata - Ruby][]
 * [Bowling Kata - Ruby][]
-* Tic-tac-toe Kata - Ruby
+* [Tic-tac-toe Kata - Ruby](https://gist.github.com/dlresende/dc57f506e1ab1de7e7df)
 
 **Training:**
 
-* Introduction to frontend project
-* [CodeSchool: Front-end Foundations][]: Level 2, Level 3, Level 5
+* Frontend Project: Introduction
```
Comment (cromwellryan) [inline index.md]: Let's split out the changes to frontend from the rest. I want to talk with @zastrow about how these should be aligned with the milestones, if at all.

File: index.md (hunk 2 above)
Comment (cromwellryan) [inline index.md]: Thanks! I'm going to go through these this week as well.

File: index.md (hunk 3 above)
Comment (cromwellryan) [inline index.md]: Remove for same reason as :point_up:

File: index.md [hunk 6]
```diff
@@ -57,36 +57,40 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
-* [CodeSchool: jQuery Air: First Flight][]
-* [CodeSchool: jQuery Air: Captain's Log][]
-* Frontend project JavaScript review (CSS to be rewritten)
-* Frontend project final review and launch!
+* [Javascript Road Trip Part I](https://www.codeschool.com/courses/javascript-road-trip-part-1)
+* [Javascript Road Trip Part II](https://www.codeschool.com/courses/javascript-road-trip-part-2)
+* [Optional: Javascript Road Trip Part III](https://www.codeschool.com/courses/javascript-road-trip-part-3)
+* Frontend Project - Review: JavaScript
+  - Rewrite your CSS :art:
+* Frontend Project: Final review & LAUNCH :rocket:
```
Comment (cromwellryan) [inline index.md]: Remove from this PR

File: index.md (hunk 4 above)
Comment (cromwellryan) [inline index.md]: 🔪

**Comment by catheraaine** [comment]
@cromwellryan updated!!

**Comment by cromwellryan** [comment]
Rebased

**Pull request merged**